newdle/client/src/components/MyNewdles.js

```diff
@@ -100,11 +101,19 @@ function MyNewdle({newdle: {code, title, participants, duration, final_dt: final
         <Icon name="users" />
         <label>
           <em>
-            {!participants.length
-              ? 'There are no participants yet'
-              : answers.length !== participants.length
-              ? `${answers.length} out of ${participants.length} participants answered`
-              : 'All participants answered'}
+            {!participants.length ? (
+              t`There are no participants yet.`
+            ) : answers.length !== participants.length ? (
+              <Trans>
+                <Plural
```

Comment:
you can add `_0="There are no participants yet."` to cover the no-participants case

Reply:
Hmm if I add that and change the logic around, you either get `All participants answered.` for 0 participants or 2 messages when everybody answered. (`x out of x answered` and `All participants answered`). To keep the display like this it's easier to leave the current logic.

Reply:
ah, i didn't realize there's yet another condition.. you are right, then it won't work... [Select](https://js-lingui-git-next.lingui-js.now.sh/ref/macro.html#jsxmacro-Select) could be an option but probably overkill here since the separate strings work fine